python/ql/src/Classes/Comparisons/Comparisons.qll

```diff
@@ -5,6 +5,8 @@ import semmle.python.ApiGraphs
 
 /** Holds if `cls` has the `functools.total_ordering` decorator. */
 predicate totalOrdering(Class cls) {
-  cls.getADecorator() =
-    API::moduleImport("functools").getMember("total_ordering").asSource().asExpr()
+  API::moduleImport("functools")
+      .getMember("total_ordering")
+      .asSource()
+      .flowsTo(DataFlow::exprNode(cls.getADecorator()))
```

Comment:
I think this is a place where we would want to use `getAValueReachableFromSource`.

Reply:
Will keep in mind for future work. I'll merge now as is.